C++/test/algorithm/dynamic_programming/longest_decreasing_subsequence.cpp

```diff
@@ -0,0 +1,13 @@
+#include "third_party/catch.hpp"
+#include "include/algorithms/dynamic_programming/longest_decreasing_subsequence.hpp"
+
+TEST_CASE("Base cases", "[longest_decreasing_subsequence]") {
+	REQUIRE(longest_decreasing_subsequence(std::vector<int>({}) == 0);
```

Comment:
You can do the same thing in this file. `using ...` at the top and then removing `std::` later on.